apache-rat-core/src/main/resources/org/apache/rat/plain-rat.xsl

```diff
@@ -19,46 +19,55 @@
 -->
 <xsl:stylesheet version="1.0"
                 xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+    <xsl:variable name='newline'><xsl:text>&#xa;</xsl:text></xsl:variable>
+
 <xsl:output method='text'/>
 <xsl:template match='/'>
 *****************************************************
 Summary
--------
+*****************************************************
```

Comment:
I assume the removed line was correct here as the asterisks indicate a start of an element? WDYT?

The report starts with asterisks and ends with asterisks.

Reply:
I just had a look at the whole report and all sections started with asterisk, whiole the ---- was just a headline.